dataactcore/models/jobModels.py

```diff
@@ -29,7 +29,7 @@ class Submission(Base):
     submission_id = Column(Integer, primary_key=True)
     datetime_utc = Column(DateTime)
     user_id = Column(Integer, nullable=False) # This refers to the users table in the User DB
-    agency_name = Column(Text)
+    cgac_code = Column(Text)
```

Comment:
I see that above we're using a surrogate key to reference back to a submission's user. Does it make sense to follow that model here and use the cgac id instead of the code?